core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java

```diff
@@ -142,30 +143,31 @@ public BlockInfo getBlockInfo(long blockId) {
    * @return a {@link Protocol.OpenUfsBlockOptions} based on the block id and options
    */
   public Protocol.OpenUfsBlockOptions getOpenUfsBlockOptions(long blockId) {
-    Preconditions.checkArgument(mStatus.getBlockIds().contains(blockId), "blockId");
-    boolean readFromUfs = mStatus.isPersisted();
+    Preconditions.checkArgument(mStatus.getBlockIds().contains(blockId),
```

Comment:
Interestingly this `getBlockIds()` returns a List, but on 2nd thought just `O(n)` scan a list is probably faster than building a set.

Reply:
Yep, the blockId list should be small for most of files